### ISSUE TITLE
"role=main should only appear on significant elements"

### ISSUE THREAD

**Issue by Charlotteis** [addyosmani/a11y#30]
Hey Team!

I just ran the a11y audit for a project I'm working on and got the following failure:
`X role=main should only appear on significant elements`

I have defined `role=main` only on my `<main></main>` element, which as far as I am aware is a fine practice to do. 

After a little googling I found the following: https://groups.google.com/forum/#!topic/introduction-to-web-accessibility/ChGzBSe0_yU but after following the fix (updating my accessibility developer tools) nothing changed unfortunately.

Any idea what the problem might be?

**Comment by brunobatista** [comment]
+1

**Comment by addyosmani** [comment]
I think the issue may be a false positive with the accessibility-developer-tools. If you're experiencing the same issue when running them directly (https://github.com/GoogleChrome/accessibility-developer-tools#command-line-runner), please file a bug there and we can explore getting it fixed upstream.

**Comment by brunobatista** [comment]
Thanks a lot @addyosmani :+1:
